edge-modules/azure-monitor/src/ExampleDeployment.json

```diff
@@ -0,0 +1,85 @@
+{
+  "modulesContent": {
+    "$edgeAgent": {
+      "properties.desired": {
+        "schemaVersion": "1.0",
+        "runtime": {
+          "type": "docker",
+          "settings": {
+            "minDockerVersion": "v1.25",
+            "loggingOptions": "",
+            "registryCredentials": {}
+          }
+        },
+        "systemModules": {
+          "edgeAgent": {
+            "type": "docker",
+            "settings": {
+              "image": "mcr.microsoft.com/azureiotedge-agent:1.0.10.1",
```

Comment:
Should by `mcr.microsoft.com/azureiotedge-agent:1.1`

Reply:
Done